keylime/tenant.py

```diff
@@ -1210,7 +1210,7 @@ def write_to_namedtempfile(data, delete_tmp_files):
     return temp.name
 
 
-def main(argv=sys.argv):  # pylint: disable=dangerous-default-value
+def main(argv: List[str] = sys.argv) -> None:  # pylint: disable=dangerous-default-value
```

Comment:
Can you remove the `argv` parameter here, the `pylint` comment, the call to `main` and in line 1224? This seems to be a mistake, and the linter is right that this is very bad

Reply:
I looked at this here: https://pylint.pycqa.org/en/latest/user_guide/messages/warning/dangerous-default-value.html

The example of a function isn't very good when applying it to `main()` that only typically gets called once. I am not sure what is so bad about it if `main()` only gets called once?

I'll change it, though.

Reply:
I think that there are two issues: (1) `argparse.ArgumentParser()` should not be called with a parameter, as by default it gets it from `sys`, and this makes the parameter `argv` superfluous, and (2) the one that you refer, that a mutable object should never be a default parameter, as can be changed causing big problems if there is a second call